ddsc/cmdparser.py

```diff
@@ -149,6 +158,17 @@ def _add_auth_role_arg(arg_parser):
                             help="Specifies authorization role for the user ('project_admin').",
                             default='project_admin')
 
+def _add_copy_project_arg(arg_parser):
```

Comment:
Because they have been used elsewhere. I like to keep everything as private as possible(start with _ in python).